ocs_ci/ocs/resources/pod.py

```diff
@@ -2547,24 +2547,51 @@ def check_toleration_on_pods(toleration_key=constants.TOLERATION_KEY):
 
     """
 
-    pod_objs = get_all_pods(
+    pod_objs = get_running_state_pods(
```

Comment:
why is this change ? if some pods in Pending state or completed state it might skip the toleration check on those pods.